package.json

```
@@ -67,7 +68,8 @@
     "webpack-cli": "^4.3.0"
   },
   "dependencies": {
-    "cafe-args": "^1.0.13",
+    "cafe-args": "^1.0.16",
```

Please use exact versioning to follow every of ur release
```suggestion
    "cafe-args": "1.0.16",
```